broker/locket.go

```diff
@@ -3,19 +3,27 @@ package broker
 import (
 	"context"
 	"sync"
+	"time"
 
 	"code.cloudfoundry.org/lager"
 	"code.cloudfoundry.org/locket"
 	locket_models "code.cloudfoundry.org/locket/models"
 	"google.golang.org/grpc"
 )
 
+var (
+	// DefaultLocketRetryInterval is the time between retries while waiting
+	// for successful locket response
```

Comment:
hooray for godoc \o/

Reply:
one down ... several hundred to go :champagne: